templates/terraform.tfvars.aws-template

```diff
@@ -4,7 +4,7 @@ cluster_prefix = "your-cluster-prefix" # Your cluster prefix
 kubeadm_token = "your-kubeadm-token" # Autogenerated kubeadm token (don't change it)
 aws_region = "eu-west-1" # Some region
 availability_zone = "eu-west-1a" # Some availability zone
-boot_image = "kubenow-v050"
+boot_image = "kubenow-v050b1"
```

Comment:
@andersla @carmat88 we should make v050 working again before moving on. Anders could you remind us what was the problem with `v050b1`, and Matteo did you rebuilt that stable making sure that it won't be updated in the future? We want start security updates from version `v051`.